docs/source/install-osx.rst

```diff
@@ -0,0 +1,49 @@
+.. _install-osx:
+
+###################
+Installation - macOS
```

Comment:
Can we add something like "macOS / OS X" for the title (we should leave it as macOS everywhere else as that's what Apple call it now, but i think having this on the first title might be useful for any users who are more familiar with the OSX name